apps/root/src/frame/index.tsx

```diff
@@ -131,7 +131,7 @@ const AppFrame = ({ config: { wagmiClient }, initialChain }: AppFrameProps) => {
                       justifyContent="center"
                       isSmall={currentBreakPoint === 'xs'}
                     >
-                      <StyledAppGridContainer item xs={12} sm={10}>
+                      <StyledAppGridContainer item xs={12} sm={10} xl={12}>
```

Comment:
The design specifies a max-width of `1160px` for big screens, this is defined in `src/frame` but never reached because of this `sm={10}` rule. I think we never needed it until now, where data cards shrink a little when they are 3 in the same row:

This is without the new `xl={12}`:
<img width="655" alt="Screenshot 2024-07-08 at 2 39 48 PM" src="https://github.com/Balmy-protocol/dca-fe/assets/144699868/a459fad5-5d3b-4d6a-a9c7-0dd10089e039">

Reply:
Yeah we already set max for each breakpoint, it should not be handled by this one

Reply:
Yeah we already set max for each breakpoint, it should not be handled by this one

Reply:
Yeah we already set max for each breakpoint, it should not be handled by this one